.github/workflows/python-package.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         poetry install
     - name: flake8 quick test
       run: |
-        # stop the build if there are Python syntax errors or undefined names
+        # stop the build if there are Python syntax errors or undefined names. TODO: this is being run in ``pre_commit_check.py`` with difference parameters. Why?
```

Comment:
This was part of a suggested workflow that I copied.  The idea is to run a very quick test looking for syntax errors up front as their is no point in going any further if the code is broken.

Reply:
I went ahead and removed this check and updated flake8 checks.